src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java

```diff
@@ -74,14 +74,18 @@ public void put(Collection<SinkRecord> records) {
           remainingRetries,
           sqle
       );
+      String sqleAllMessages = "";
+      for (Throwable e : sqle) {
+        sqleAllMessages += e + System.lineSeparator();
+      }
       if (remainingRetries == 0) {
-        throw new ConnectException(sqle);
+        throw new ConnectException(SQLException(sqleAllMessages));
```

Comment:
Not sure if we didn't notice before, or this change happened along with rebasing, but this leads to compilation error: 
SQLException needs construction with `new`

Reply:
Yeah that was me trying to fix the unit test. Forgot the new keyword. Fixed now. And looks like tests passed.